src/js/components/Paragraph/Paragraph.js

```diff
@@ -26,9 +26,11 @@ const Paragraph = forwardRef(
         size={size}
         {...rest}
       >
-        <TextContext.Provider value={textContextValue}>
-          {children}
-        </TextContext.Provider>
+        {children !== undefined ? (
+          <TextContext.Provider value={textContextValue}>
+            {children}
+          </TextContext.Provider>
+        ) : undefined}
```

Comment:
Just a suggestion for a slight simplification
```suggestion
        {children && (
          <TextContext.Provider value={textContextValue}>
            {children}
          </TextContext.Provider>
        )}
```

Reply:
@jcfilben I believe it would be best to avoid this simplification, as it could cause issues if a value of 0 is passed as children to the component. This is because `0 && X` evaluates to `false`, which would prevent the text from rendering properly. What are your thoughts on this?

Reply:
I'm okay with leaving this as it